libafl_bolts/src/llmp.rs

```diff
@@ -455,7 +463,7 @@ fn tcp_bind(port: u16) -> Result<TcpListener, Error> {
 
 /// Send one message as `u32` len and `[u8;len]` bytes
 #[cfg(feature = "std")]
-fn send_tcp_msg<T>(stream: &mut TcpStream, msg: &T) -> Result<(), Error>
+pub fn send_tcp_msg<T>(stream: &mut TcpStream, msg: &T) -> Result<(), Error>
```

Comment:
Would `pub(crate)` be enough?

Reply:
no. because i have to use it from libafl not libafl_bolts